CMakeLists.txt

```diff
@@ -119,6 +119,10 @@ add_cts_option(SYCL_CTS_ENABLE_KHR_DEFAULT_CONTEXT_TESTS
     "Enable extension Khronos default_context tests" OFF
     FORCE_ON ${SYCL_CTS_ENABLE_KHR_TESTS})
 
+add_cts_option(SYCL_CTS_ENABLE_KHR_FREE_FUNCTION_COMMANDS
```

Comment:
It looks like according to the neighborhood this should be named `SYCL_CTS_ENABLE_KHR_FREE_FUNCTION_COMMANDS_TESTS`, as mentioned in SYCL_CTS_ENABLE_KHR_FREE_FUNCTION_COMMANDS